app/src/views/OperationalLearnings/index.tsx

```diff
@@ -1,17 +1,195 @@
+    const [tabValue, setTabValue] = useState('by-sector');
+
+    const handleTabChange = (event: React.MouseEvent<HTMLButtonElement>) => {
+        const { name } = event.currentTarget;
+        setTabValue(name);
+    };
+
+    function handleExportClick(): void {
+        // Export functionality here
+    }
+
     return (
         <Page
             heading={strings.operationalLearningsHeading}
             description={strings.operationalLearningsHeadingDescription}
-        />
+        >
+            <Container
+                className={styles.operationalLearnings}
+                contentViewType="vertical"
+                withGridViewInFilter
+                filters={(
+                    <>
+                        <RegionSelectInput
+                            placeholder={strings.filterRegionsPlaceholder}
+                            name="Regions"
+                            value={regionValue ?? null}
+                            onChange={onchange}
+                        />
+                        <CountrySelectInput
+                            placeholder={strings.filterCountryPlaceholder}
+                            name="Country"
+                            value={countryValue}
+                            onChange={onchange}
+                        />
+                        <DisasterTypeSelectInput
+                            placeholder={strings.filterDisasterTypePlaceholder}
+                            name="Disaster type"
+                            value={disasterTypeValue}
+                            onChange={onchange}
+                        />
+                        <SelectInput
+                            placeholder={strings.filterBySectorPlaceholder}
+                            name="by sector"
+                            options={options}
+                            keySelector={keySelector}
+                            labelSelector={labelSelector}
+                            value={sectorValue}
+                            onChange={() => {onchange
+                            }}
+                        />
+                        <SelectInput
+                            placeholder={strings.filterByComponentPlaceholder}
+                            name="by component"
+                            options={options}
+                            keySelector={keySelector}
+                            labelSelector={labelSelector}
+                            value={componentValue}
+                            onChange={() => {onchange
+                            }}
+                        />
+                        <DateInput
+                            name="dateFrom"
+                            value={null}
+                            onChange={() => {onchange
+                            }}
+                        />
+                        <div>
+                            <KeywordSearchSelectInput />
+                        </div>
```

Comment:
Why are we  wrapping the component with a `div`.  We should only add elements when they are necessary.